care/facility/api/viewsets/patient.py

```diff
@@ -604,13 +609,18 @@ def list(self, request, *args, **kwargs):
 
 
 class PatientNotesViewSet(
-    ListModelMixin, RetrieveModelMixin, CreateModelMixin, GenericViewSet
+    ListModelMixin,
+    RetrieveModelMixin,
+    CreateModelMixin,
+    UpdateModelMixin,
+    GenericViewSet,
 ):
     queryset = (
         PatientNotes.objects.all()
         .select_related("facility", "patient", "created_by")
         .order_by("-created_date")
     )
+    http_method_names = ["get", "post", "put", "patch"]
```

Comment:
Why do you need to state this?

Reply:
You're right. This is inferred from the `PatientNotesViewSet` Mixin. Removed.